File: javascript/7-kyu/shortest-word.js
// Simple, given a string of words, return the length of the shortest word(s).

// String will never be empty and you do not need to account for different data types.

function findShort(s) {
  let l = 69
  let array = s.split(" ")
  for (i = 0; i < array.length; i++) {
    if (array[i].length < l) {
      l = array[i].length
    } 
  }
  return l

File: python/7-kyu/shortest-word.py
# Simple, given a string of words, return the length of the shortest word(s).

# String will never be empty and you do not need to account for different data types.

def find_short(s):
    x = "alskdfoisjdfoapisjdfoisjodfijsodifjsodifjosidfjosdifjosdifjosidjfsdoif"
    lst = s.split()
    for word in lst:
        if len(word) < len(x):
            x = word
    return len(x)
